grass7/vector/v.rast.bufferstats/v.rast.bufferstats.py

```diff
@@ -98,16 +99,32 @@
 #% required : no
 #%end
 
+#%option
+#% key: resolution
+#% type: integer
+#% description: The region resolution to get better result in the analysis
```

Comment:
This description is not very clear yet

Reply:
The original idea was to use and align to the resolution of the current region... If the module dos not follow the current region I would prefer to change it in a way that it does...

Reply:
> The original idea was to use and align to the resolution of the current region... If the module dos not follow the current region I would prefer to change it in a way that it does...

Ok, I'm going to remove this feature